recipes/foxglove-websocket/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_package test_package.cpp)
+target_link_libraries(test_package ${CONAN_LIBS})
```

Comment:
Looking quickly you'll need to configure C++17 here ... here's an example https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/libsolace/all/test_package/CMakeLists.txt#L12

I'll take a better look tomorrow of theres anything else I can find

Reply:
I think that helped for setting the language standard but still running into problems with the standard library version.